### PR TITLE
showcase RC testing

### PR DESCRIPTION
**Pull request by MorrisJobke**
* based on #8 
* this worked for me locally here

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @nickvergessen @LukasReschke

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+vendor/behat
+composer.lock
```
Comment (nickvergessen) [inline .gitignore]: Thanks ;)

File: config/config.php [hunk 2]
```diff
@@ -104,8 +104,9 @@
 	],
 	'beta' => [
 		'9.1' => [
-			'latest' => '10.0.0',
-			'internalVersion' => '9.1.0',
+			'latest' => '10.0.1RC1',
+			'internalVersion' => '9.1.1RC1',
+			'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-10.0.1RC1.zip',
```
Comment (nickvergessen) [inline config/config.php]: Could also add magic to the url generation, if `RC` ist part of the version

Comment (MorrisJobke) [inline config/config.php]: Fewer logic is nice for now. We could add this later.

**Pull request merged**